src/commands/support/supportGuide.ts

```diff
@@ -11,7 +11,7 @@ const supportGuideEmbed = makeEmbed({
         '- Be sure you know how to fly the A32NX by reading our [beginners guide](https://docs.flybywiresim.com/pilots-corner/beginner-guide/overview/).',
         '- Check our [reported issues list](https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/) to see if your issue has a known solution.',
         '- Verify the issue is reproducible with only the A32NX in the community folder and no external programs affecting MSFS are running. For example: GSX, SPAD.neXt, ...',
-        '- Clearly document the steps to reproduce the issue and provide them together with the screenshots in the #a32nx-support channel.',
+        '- Clearly document the steps to reproduce the issue and provide them together with the screenshots in the [#a32nx-support](https://discord.com/channels/738864299392630914/785976111875751956) channel.',
```

Comment:
```suggestion
        '- Clearly document the steps to reproduce the issue and provide them together with the screenshots in the <#${Channels.A32NX_SUPPORT}> channel.',
```

I would tend to use this instead of the link